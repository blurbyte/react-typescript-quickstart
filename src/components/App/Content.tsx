import styled from '@styles/styledComponents';

const Content = styled.section`
  display: grid;
  grid-template-columns: repeat(3, 1fr);
  grid-gap: 20px;
`;

export default Content;
