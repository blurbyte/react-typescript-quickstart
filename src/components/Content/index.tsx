import styled from '@styles/styledComponents';
import media from '@styles/media';

const Content = styled.div`
  padding: 0 6.4rem;
  max-width: 144rem;
  margin: 0 auto;

  ${media.desktop`
    padding: 0 3.2rem;
  `};
`;

export default Content;
