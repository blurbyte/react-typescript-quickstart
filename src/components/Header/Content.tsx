import styled from '@styles/styledComponents';

import BaseContent from '../Content';

const Content = styled(BaseContent)`
  display: flex;
  align-items: center;
  height: 6.4rem;
`;

export default Content;
