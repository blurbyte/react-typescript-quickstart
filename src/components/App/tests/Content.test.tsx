import * as React from 'react';
import { shallow } from 'enzyme';

import Content from '../Content';

describe('Wrapper', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Content />);
    expect(wrapper).toMatchSnapshot();
  });
});
