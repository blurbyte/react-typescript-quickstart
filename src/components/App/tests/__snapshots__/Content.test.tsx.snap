// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: repeat(3,1fr);
  grid-gap: 20px;
}

<section
  className="c0"
/>
`;
